configs/apalis-imx8_defconfig

```diff
@@ -1,16 +1,31 @@
 CONFIG_ARM=y
+CONFIG_SPL_SYS_ICACHE_OFF=y
+CONFIG_SPL_SYS_DCACHE_OFF=y
 CONFIG_ARCH_IMX8=y
+CONFIG_SPL_GPIO_SUPPORT=y
+CONFIG_SPL_LIBCOMMON_SUPPORT=y
+CONFIG_SPL_LIBGENERIC_SUPPORT=y
 CONFIG_SYS_TEXT_BASE=0x80020000
 CONFIG_SYS_MALLOC_F_LEN=0x8000
 CONFIG_ENV_SIZE=0x2000
 CONFIG_ENV_OFFSET=0xFFFFDE00
 CONFIG_DM_GPIO=y
+CONFIG_SPL_LOAD_IMX_CONTAINER=y
```

Comment:
Are we sure we need this?
```
config SPL_LOAD_IMX_CONTAINER
        bool "Enable SPL loading U-Boot as a i.MX Container image"
        depends on SPL
        help
          This is to let SPL could load i.MX8 Container image
```

Reply:
(Discussion from Slack: this is the "default" defconfig.  We disable this for LmP where we use u-boot.itb)